### ISSUE TITLE
Disable state of VBO button has poor contrast

### ISSUE THREAD

**Issue by CraigClark** [bcgov/MFIN-Data-Catalogue#114]
**OP timer**

https://openplus.monday.com/boards/4092908516/pulses/4966205903

---

## Describe the bug

The disabled state of the VBO on the content overview page does not pass a11y.

## To reproduce
Steps to reproduce the behaviour:
1. Go to /admin/content
2. Select a checkbox beside a piece of content
3. Scroll down to the vbo actions
4. See that the contrast ratio is 1.12 (fail)

## Expected behaviour
Visually impaired people can read the text, even though the button is disabled.

## Screenshots
![disabled button has contrast ratio of contrast ratio is 1.12 (fail)](https://github.com/bcgov/MFIN-Data-Catalogue/assets/3910508/45492104-2397-425e-ad59-67d2b07011ee)

## Additional context

If you use `#828388` for the text and `#000071` for the background, it will have a ratio of 4.5 which passes for text 14pt+
